### PR TITLE
taos-connector-python merge main to 3.0 and modify stmt2_get_fields interface 

### PR DESCRIPTION
**Pull request by DuanKuanJun**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -7,6 +7,231 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
...
+## v2.7.18 - 2024-12-14
+
+### Features:
+
+-  support taos taosws taosrest protocol for superset
+
+### Bug Fixes:
+
+-  2.x add taos-ws-py compose
+-  DATABASE_URL KeyError
```
Comment (sheyanjie-qq) [inline CHANGELOG.md:19]: we need rewrite this list, do not use the git log directly.

File: CHANGELOG.md [hunk 2]
```diff
@@ -7,6 +7,231 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
...
+
+### Documents:
+
+-  optimize taospy documents on PyPi official website
+
+## v2.7.17 - 2024-12-14
+
+## v2.7.17 - 2024-12-14
+
+### Features:
```
Comment (sheyanjie-qq) [inline CHANGELOG.md:176]: duplicated info in  v2.7.17 and v2.7.18

File: taos-ws-py/CHANGELOG.md [hunk 3]
```diff
@@ -7,6 +7,33 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
...
+## v0.3.4 - 2024-11-19
+
+## v0.3.4 - 2024-11-19
+
+## v0.3.4 - 2024-11-19
+
+## v0.3.4 - 2024-11-19
+
+## v0.3.4 - 2024-11-19
+
```
Comment (sheyanjie-qq) [inline taos-ws-py/CHANGELOG.md:19]: duplicated info

File: taos-ws-py/tests/test_cloud.py [hunk 4]
```diff
@@ -0,0 +1,13 @@
+import taosws
+
+def test_ws_connect():
+    print("-" * 40)
+    print("test_ws_connect")
+    conn = taosws.connect("%s?token=%s" % ('wss://gw.cloud.taosdata.com', '68d41c020d2e9e3dcac18140460d1bcfd82d642d'))
```
Comment (sheyanjie-qq) [inline taos-ws-py/tests/test_cloud.py:6]: do not use token in code, use env variable

File: CHANGELOG.md (hunk 2 above)
Comment (DuanKuanJun) [reply CHANGELOG.md:176]: already remove duplicated info

File: taos-ws-py/tests/test_cloud.py (hunk 4 above)
Comment (DuanKuanJun) [reply taos-ws-py/tests/test_cloud.py:6]: already remove duplicated info

File: taos-ws-py/CHANGELOG.md (hunk 3 above)
Comment (DuanKuanJun) [reply taos-ws-py/CHANGELOG.md:19]: already remove duplicated info

File: CHANGELOG.md (hunk 1 above)
Comment (DuanKuanJun) [reply CHANGELOG.md:19]: rewrite  manually

**Review by sheyanjie-qq — APPROVED** [review]

**Pull request merged**